data-raw/migratory_corridor/migratory_temps.R

```diff
@@ -154,6 +156,16 @@ migratory_temperature_proportion_over_20 <- median_p20 %>%
   spread(month, median_p20) %>%
   select(`1`:`12`) %>%
   as.matrix()
-dimnames(migratory_temperature_proportion_over_20) <- list(cvpiaFlow::watershed_ordering$watershed, month.abb)
+dimnames(migratory_temperature_proportion_over_20) <- list(watersheds_order$watershed, month.abb)
 
 usethis::use_data(migratory_temperature_proportion_over_20, overwrite = TRUE)
+
+# migratory for springRun
+## update to match Tuolumne River (30)
+migratory_temperature_proportion_over_20_sr <- migratory_temperature_proportion_over_20
+migratory_temperature_proportion_over_20_sr[31, ] <- migratory_temperature_proportion_over_20_sr[30, ]
```

Comment:
@ErinCain this is the updated logic. 

`migratory_temperature_proportion_over_20` gets built and then I replace `San Joaquin River` with values from `Tuolumne River`. This logic was vetted by previously running `migratory_temperature_proportion_over_20` without SJ in `missing`